packages/app/components/DatePicker.tsx

```diff
@@ -0,0 +1,170 @@
+"use client";
+
+import { format, isToday, isTomorrow } from "date-fns";
+
+import { Button, Calendar, Icon } from "@/ui";
+import { Popover } from "@headlessui/react";
+
+import { Dispatch, Ref, SetStateAction, useEffect, useState } from "react";
+import { usePopper } from "react-popper";
+import { twMerge } from "tailwind-merge";
+
+interface IDatePicker {
```

Comment:
When interface is the props the component receives I'm using `DatePickerProps` naming convention. 

```suggestion
interface DatePickerProps {
```

Don't know if we want to change it to this convention, but think we should stick with one.

Reply:
Both work to me, let's use `DatePickerProps`